content/docs/12.5-vertex/02-install-palette-vertex/10-install-on-vmware.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Install on VMware vSphere"
+metaTitle: "Install on VMware vSphere"
+metaDescription: "Learn how to install Palette on VMware vSphere."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Install Palette VerteX on VMware vSphere
+
+You install Palette VerteX using a Command Line Interface (CLI) that guides you for details. The installer creates a three-node enterprise cluster for high availability (HA): 6 vCPU, 12 GB memory, 70 GB storage. You can invoke the installer on any Linux x86-64 system that has the docker daemon installed and connectivity to the Palette Management console or VMware vSphere. 
```

Comment:
```suggestion
You install Palette VerteX using a Command Line Interface (CLI) that guides you for details. The installer creates a three-node enterprise cluster for high availability (HA): 6 vCPU, 12 GB memory, 70 GB storage. You can invoke the installer on any Linux x86-64 system that has the docker daemon installed and connectivity to the Palette Management console and to the VMware vSphere where Palette VerteX should be deployed to. 
```

Questions:
- are you sure that the CLI deploys a 3-nodes cluster? IIRC, it deploys a single node that scale out to 3 nodes after, no?
- "6 vCPU, 12 GB memory, 70 GB storage": is that for each node? or is it the sum of all node resources?